module,execution_time (sec),success,timestamp,error,error_short
cowidev.vax.batch.argentina,4.65,True,2022-08-22T07:00:16,,
cowidev.vax.batch.australia,4.5,True,2022-08-22T07:00:16,,
cowidev.vax.batch.austria,4.58,True,2022-08-22T07:00:16,,
cowidev.vax.batch.belgium,2.27,True,2022-08-22T07:00:13,,
cowidev.vax.batch.bolivia,2.95,True,2022-08-22T07:00:14,,
cowidev.vax.batch.canada,11.62,False,2022-08-22T07:04:46,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 255, in main
    Canada().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 242, in export
    df_man = df_man.pipe(self.pipeline_manufacturer)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/core/generic.py"", line 5430, in pipe
    return com.pipe(self, func, *args, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/pandas/core/common.py"", line 471, in pipe
    return func(obj, *args, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/batch/canada.py"", line 173, in pipeline_manufacturer
    validate_vaccines(df, self.vaccine_mapping)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/utils/checks.py"", line 258, in validate_vaccines
    raise ValueError(f""Missing vaccines: {vaccines_wrong}"")
ValueError: Missing vaccines: {'Pfizer-BioNTech Comirnaty (ages 5-11 years)', 'Novavax Nuvaxovid', 'Moderna Spikevax (ages 6 months-5 years)', 'Pfizer-BioNTech Comirnaty (ages 12 years and older)'}
","Missing vaccines: {'Pfizer-BioNTech Comirnaty (ages 5-11 years)', 'Novavax Nuvaxovid', 'Moderna Spikevax (ages 6 months-5 years)', 'Pfizer-BioNTech Comirnaty (ages 12 years and older)'}"
cowidev.vax.batch.chile,1.83,True,2022-08-22T07:00:16,,
cowidev.vax.batch.czechia,2.78,True,2022-08-22T07:00:18,,
cowidev.vax.batch.denmark,3.26,True,2022-08-22T07:00:19,,
cowidev.vax.batch.ecdc,152.54,True,2022-08-22T07:02:48,,
cowidev.vax.batch.ecuador,2.09,True,2022-08-22T07:00:18,,
cowidev.vax.batch.estonia,7.77,True,2022-08-22T07:00:26,,
cowidev.vax.batch.france,1.28,True,2022-08-22T07:00:20,,
cowidev.vax.batch.germany,1.67,True,2022-08-22T07:00:21,,
cowidev.vax.batch.greece,3.43,True,2022-08-22T07:00:23,,
cowidev.vax.batch.hong_kong,5.51,True,2022-08-22T07:00:26,,
cowidev.vax.batch.indonesia,,,2022-08-22T07:00:23,,
cowidev.vax.batch.ireland,0.35,True,2022-08-22T07:00:23,,
cowidev.vax.batch.israel,10.96,True,2022-08-22T07:00:34,,
cowidev.vax.batch.italy,9.25,True,2022-08-22T07:00:34,,
cowidev.vax.batch.japan,131.53,True,2022-08-22T07:02:37,,
cowidev.vax.batch.jersey,8.07,True,2022-08-22T07:00:34,,
cowidev.vax.batch.latvia,17.92,True,2022-08-22T07:00:52,,
cowidev.vax.batch.lithuania,1.93,True,2022-08-22T07:00:36,,
cowidev.vax.batch.luxembourg,2.55,True,2022-08-22T07:00:37,,
cowidev.vax.batch.malaysia,0.64,True,2022-08-22T07:00:37,,
cowidev.vax.batch.malta,1.52,True,2022-08-22T07:00:38,,
cowidev.vax.batch.netherlands,,,2022-08-22T07:00:37,,
cowidev.vax.batch.new_zealand,11.4,True,2022-08-22T07:00:48,,
cowidev.vax.batch.norway,0.56,True,2022-08-22T07:00:39,,
cowidev.vax.batch.peru,3.37,True,2022-08-22T07:00:42,,
cowidev.vax.batch.portugal,,,2022-08-22T07:00:42,,
cowidev.vax.batch.romania,15.41,True,2022-08-22T07:00:58,,
cowidev.vax.batch.saudi_arabia,3.28,True,2022-08-22T07:00:52,,
cowidev.vax.batch.singapore,3.39,True,2022-08-22T07:00:55,,
cowidev.vax.batch.slovakia,111.63,True,2022-08-22T07:02:44,,
cowidev.vax.batch.slovenia,2.33,True,2022-08-22T07:00:57,,
cowidev.vax.batch.south_korea,68.59,True,2022-08-22T07:02:06,,
cowidev.vax.batch.spc,21.41,True,2022-08-22T07:01:19,,
cowidev.vax.batch.sweden,32.93,True,2022-08-22T07:01:52,,
cowidev.vax.batch.switzerland,54.7,True,2022-08-22T07:02:47,,
cowidev.vax.batch.trinidad_and_tobago,2.18,True,2022-08-22T07:02:08,,
cowidev.vax.batch.ukraine,,,2022-08-22T07:02:08,,
cowidev.vax.batch.united_kingdom,18.86,True,2022-08-22T07:02:27,,
cowidev.vax.batch.united_states,26.91,True,2022-08-22T07:02:54,,
cowidev.vax.batch.uruguay,7.71,True,2022-08-22T07:02:45,,
cowidev.vax.batch.zimbabwe,0.38,True,2022-08-22T07:02:44,,
cowidev.vax.incremental.africacdc,,,2022-08-22T07:02:44,,
cowidev.vax.incremental.antigua_barbuda,3.81,True,2022-08-22T07:02:48,,
cowidev.vax.incremental.aruba,2.55,True,2022-08-22T07:02:47,,
cowidev.vax.incremental.azerbaijan,70.11,True,2022-08-22T07:03:57,,
cowidev.vax.incremental.bahrain,3.5,True,2022-08-22T07:02:51,,
cowidev.vax.incremental.bangladesh,21.55,True,2022-08-22T07:03:09,,
cowidev.vax.incremental.barbados,1.58,True,2022-08-22T07:02:50,,
cowidev.vax.incremental.brazil,1.19,True,2022-08-22T07:02:51,,
cowidev.vax.incremental.bulgaria,1.18,True,2022-08-22T07:02:52,,
cowidev.vax.incremental.china,41.56,True,2022-08-22T07:03:33,,
cowidev.vax.incremental.costa_rica,,,2022-08-22T07:02:52,,
cowidev.vax.incremental.croatia,1.2,True,2022-08-22T07:02:53,,
cowidev.vax.incremental.cuba,1.96,True,2022-08-22T07:02:55,,
cowidev.vax.incremental.curacao,0.35,True,2022-08-22T07:02:54,,
cowidev.vax.incremental.cyprus,2.24,True,2022-08-22T07:02:56,,
cowidev.vax.incremental.dominican_republic,7.21,True,2022-08-22T07:03:02,,
cowidev.vax.incremental.el_salvador,40.95,False,2022-08-22T07:05:27,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 445, in _make_request
    six.raise_from(e, None)
  File ""<string>"", line 3, in raise_from
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 440, in _make_request
    httplib_response = conn.getresponse()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 1349, in getresponse
    response.begin()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 316, in begin
    version, status, reason = self._read_status()
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/http/client.py"", line 277, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), ""iso-8859-1"")
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/socket.py"", line 704, in readinto
    return self._sock.recv_into(b)
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/ssl.py"", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File ""/home/owid/.pyenv/versions/3.9.6/lib/python3.9/ssl.py"", line 1099, in read
    return self._sslobj.read(len, buffer)
socket.timeout: The read operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/adapters.py"", line 440, in send
    resp = conn.urlopen(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 755, in urlopen
    retries = retries.increment(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/util/retry.py"", line 532, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/packages/six.py"", line 770, in reraise
    raise value
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 699, in urlopen
    httplib_response = self._make_request(
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 447, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/urllib3/connectionpool.py"", line 336, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='covid19.gob.sv', port=443): Read timed out. (read timeout=20)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/el_salvador.py"", line 136, in main
    ElSalvador().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/el_salvador.py"", line 122, in export
    data = self.read().pipe(self.pipeline)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/el_salvador.py"", line 16, in read
    soup = get_soup(self.source_url)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 83, in get_soup
    response = get_response(source, request_method, use_proxy, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 49, in get_response
    raise err
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 43, in get_response
    response = requests.get(source, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/api.py"", line 75, in get
    return request('get', url, params=params, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/api.py"", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/sessions.py"", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/sessions.py"", line 645, in send
    r = adapter.send(request, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/requests/adapters.py"", line 532, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='covid19.gob.sv', port=443): Read timed out. (read timeout=20)
","HTTPSConnectionPool(host='covid19.gob.sv', port=443): Read timed out. (read timeout=20)"
cowidev.vax.incremental.faeroe_islands,0.39,True,2022-08-22T07:03:03,,
cowidev.vax.incremental.fiji,0.0,True,2022-08-22T07:03:03,,
cowidev.vax.incremental.finland,0.25,True,2022-08-22T07:03:03,,
cowidev.vax.incremental.gabon,,,2022-08-22T07:03:03,,
cowidev.vax.incremental.georgia,0.58,True,2022-08-22T07:03:04,,
cowidev.vax.incremental.greenland,1.01,True,2022-08-22T07:03:05,,
cowidev.vax.incremental.guatemala,,,2022-08-22T07:03:05,,
cowidev.vax.incremental.guernsey,0.12,True,2022-08-22T07:03:05,,
cowidev.vax.incremental.hungary,7.51,True,2022-08-22T07:03:12,,
cowidev.vax.incremental.iceland,0.5,True,2022-08-22T07:03:10,,
cowidev.vax.incremental.india,0.87,True,2022-08-22T07:03:11,,
cowidev.vax.incremental.iran,3.82,True,2022-08-22T07:03:15,,
cowidev.vax.incremental.isle_of_man,0.4,True,2022-08-22T07:03:13,,
cowidev.vax.incremental.jamaica,3.08,True,2022-08-22T07:03:16,,
cowidev.vax.incremental.kazakhstan,9.73,True,2022-08-22T07:03:24,,
cowidev.vax.incremental.kosovo,2.65,True,2022-08-22T07:03:18,,
cowidev.vax.incremental.kyrgyzstan,2.0,True,2022-08-22T07:03:20,,
cowidev.vax.incremental.laos,3.03,True,2022-08-22T07:03:24,,
cowidev.vax.incremental.macao,15.34,True,2022-08-22T07:03:39,,
cowidev.vax.incremental.moldova,0.92,True,2022-08-22T07:03:25,,
cowidev.vax.incremental.monaco,47.03,True,2022-08-22T07:04:12,,
cowidev.vax.incremental.mongolia,0.82,True,2022-08-22T07:03:33,,
cowidev.vax.incremental.montenegro,0.09,True,2022-08-22T07:03:33,,
cowidev.vax.incremental.morocco,,,2022-08-22T07:03:33,,
cowidev.vax.incremental.myanmar,,,2022-08-22T07:03:33,,
cowidev.vax.incremental.nepal,,,2022-08-22T07:03:33,,
cowidev.vax.incremental.north_macedonia,,,2022-08-22T07:03:33,,
cowidev.vax.incremental.northern_cyprus,1.13,True,2022-08-22T07:03:35,,
cowidev.vax.incremental.paho,60.09,True,2022-08-22T07:04:35,,
cowidev.vax.incremental.pakistan,1.49,True,2022-08-22T07:03:38,,
cowidev.vax.incremental.philippines,,,2022-08-22T07:03:38,,
cowidev.vax.incremental.poland,0.12,True,2022-08-22T07:03:38,,
cowidev.vax.incremental.qatar,,,2022-08-22T07:03:38,,
cowidev.vax.incremental.russia,2.17,False,2022-08-22T07:05:29,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 80, in main
    Russia().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 66, in export
    data = read(source).pipe(pipeline)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/russia.py"", line 11, in read
    soup = get_soup(source)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 83, in get_soup
    response = get_response(source, request_method, use_proxy, **kwargs)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/utils/web/scraping.py"", line 52, in get_response
    raise ValueError(
ValueError: Source https://gogov.ru/articles/covid-v-stats not reached! Error code 502 Bad Gateway
",Source https://gogov.ru/articles/covid-v-stats not reached! Error code 502 Bad Gateway
cowidev.vax.incremental.saint_lucia,,,2022-08-22T07:03:39,,
cowidev.vax.incremental.serbia,7.45,True,2022-08-22T07:03:46,,
cowidev.vax.incremental.south_africa,9.14,True,2022-08-22T07:03:48,,
cowidev.vax.incremental.spain,7.41,True,2022-08-22T07:03:54,,
cowidev.vax.incremental.sri_lanka,,,2022-08-22T07:03:48,,
cowidev.vax.incremental.suriname,17.46,True,2022-08-22T07:04:06,,
cowidev.vax.incremental.taiwan,14.78,True,2022-08-22T07:04:09,,
cowidev.vax.incremental.thailand,6.88,False,2022-08-22T07:05:36,"Traceback (most recent call last):
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/cmd/commons/get.py"", line 47, in run
    module.main()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/thailand.py"", line 62, in main
    Thailand().export()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/thailand.py"", line 56, in export
    df = self.read()
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/thailand.py"", line 15, in read
    df = self._parse_data(self.source_url)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/src/cowidev/vax/incremental/thailand.py"", line 23, in _parse_data
    return ts.getWorksheet(""D_Vac_Stack"").data
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/tableauscraper/TableauScraper.py"", line 109, in getWorksheet
    return dashboard.getWorksheet(self, self.data, self.info, worksheetName)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/tableauscraper/dashboard.py"", line 33, in getWorksheet
    indicesInfo = utils.getIndicesInfo(presModelMap, worksheet)
  File ""/mnt/owid_live_covid/covid-19-data/scripts/venv/lib/python3.9/site-packages/tableauscraper/utils.py"", line 128, in getIndicesInfo
    genVizDataPresModel = presModelMap[""vizData""][
KeyError: 'D_Vac_Stack'
",'D_Vac_Stack'
cowidev.vax.incremental.turkey,0.6,True,2022-08-22T07:04:04,,
cowidev.vax.incremental.united_arab_emirates,,,2022-08-22T07:04:04,,
cowidev.vax.incremental.vietnam,,,2022-08-22T07:04:04,,
cowidev.vax.incremental.who,6.19,True,2022-08-22T07:04:10,,
cowidev.vax.incremental.zambia,0.91,True,2022-08-22T07:04:07,,
